src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingForm.tsx

```diff
@@ -38,60 +40,55 @@ class ReprocessingForm extends React.Component<Props> {
           <FeatureBadge type="alpha" />
         </Header>
         <Body>
-          <Alert type="warning">
-            {t(
-              'Reprocessing is a preview feature. Please carefully review the limitations and implications!'
-            )}
-          </Alert>
```

Comment:
Can we bring this alert back in some form? I would like this dialog to stick out so that it's 100% clear that this is an unstable feature.